From 0ac99c0f2b5d1b86c1cd155c69e2a3fd86a0d448 Mon Sep 17 00:00:00 2001
From: lelgenio <lelgenio@disroot.org>
Date: Fri, 18 Aug 2023 14:15:35 -0300
Subject: [PATCH 3/3] don't check for new versions

---
 dzgui.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dzgui.sh b/dzgui.sh
index 893851d..ec072e4 100755
--- a/dzgui.sh
+++ b/dzgui.sh
@@ -1891,7 +1891,7 @@ initial_setup(){
 	run_depcheck
 	watcher_deps
 	check_architecture
-	check_version
+	# check_version
 	# check_map_count
 	fetch_helpers
 	config
-- 
2.40.1

