diff --git a/dzgui.sh b/dzgui.sh
index 97b5cd0..5e661cc 100755
--- a/dzgui.sh
+++ b/dzgui.sh
@@ -1056,6 +1056,10 @@ console_dl(){
 	local wid=$(xdotool search --onlyvisible --name Steam)
 	#xdotool windowactivate $wid
 	sleep 1.5s
+	xdotool type --delay 0 "noop"
+	sleep 0.5s
+	xdotool key --window $wid Return
+	sleep 0.5s
 	for i in "${modids[@]}"; do
 		xdotool type --delay 0 "workshop_download_item $aid $i"
 		sleep 0.5s
